specifyweb/frontend/js_src/lib/components/AttachmentsBulkImport/ViewAttachmentFiles.tsx

```diff
@@ -120,8 +118,8 @@ export function ViewAttachmentFiles({
   );
   const headers = React.useMemo(
     () => ({
-      selectedFileName: commonText.selectedFileName(),
-      fileSize: attachmentsText.fileSize(),
+      selectedFileName: <>{commonText.selectedFileName()}</>,
+      fileSize: <>{attachmentsText.fileSize()}</>,
```

Comment:
I left a comment earlier saying that `headerElements` prop in `GenericSortedDataViewer` shouldn't exist - why do you have it? (isn't `headers` prop enough?)

if you don't have it, then this change with `<>` `</>` is not needed as the types support both LocalizedString and JSX.Element

Reply:
The reason I needed it is because there is that schema viewer also has sort config buttons, which we don't support right now in attachment dataset viewer. Still refactored to get rid of `headerElements`.